interfaces/builtin/pulseaudio.go

```diff
@@ -23,27 +23,157 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
-/etc/pulse/ r,
-/etc/pulse/* r,
+var pulseaudioConnectedPlugAppArmor = []byte(`
+capability dac_override,
+
+/etc/machine-id r,
```

Comment:
It uses machine-id in various places to name its sockets. On the client side its using this to build the correct socket name which it will then access.